groups/orchestrator.go

```diff
@@ -171,7 +172,17 @@ func prepareJob(ctx context.Context, t *templates_v1.InstanceTemplate, group *Se
 		return nil, err
 	}
 
-	jobT := template.Must(template.New("job").Parse(jobTemplate))
+	funcMap := template.FuncMap{
+		"base64_encode": func(s string) string {
```

Comment:
Any chance we could move these template functions out of the function they're currently in, and run a few unit tests against them?

Reply:
I will.

Albeit since these template functions use part of the standard library and are single line functions (which just pass a value through to the concrete implementation; there isn't any logic in it and/or any auxiliary implementation), it might be an overkill to have an unit test for it as these are well tested already.